### PR TITLE
Flares notebook

### PR DESCRIPTION
**Pull request by arjunsavel**

### CODE REVIEW

File: interm_notebook_5_classify.ipynb [hunk 1]
```diff
@@ -0,0 +1,1175 @@
+{
```
Comment (ttdu) [inline interm_notebook_5_classify.ipynb:1]: <p>I see there are a few "to be linked" on this page. Here are the links we typically use for "<a href="https://archive.stsci.edu/missions-and-data/tess" target="_blank">TESS mission</a>" and "<a href="https://spacetelescope.github.io/hellouniverse/intro.html" target="_blank">Hello Universe</a>".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (ttdu) [inline interm_notebook_5_classify.ipynb:1]: <p>I think we talked about the "quick and dirty" trick of </p><pre class="ql-syntax" spellcheck="false">conda run -n tess pip install [package]
</pre><p>This method won't persist, but it's a good way to get a package running quickly. We can also link to the "real" methods of handling packages: <a href="https://outerspace.stsci.edu/display/MASTDOCS/Managing+Software" target="_blank">Managing+Software</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (ttdu) [inline interm_notebook_5_classify.ipynb:1]: <p>These two paragraphs are at the right level of detail (don't go into the physical motivations of flare shapes). I think some external links would be good here (e.g. to an explanation of gradient descent)</p><p> </p><p>Describing CNNs here would be the right move</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (ttdu) [inline interm_notebook_5_classify.ipynb:1]: <p>This is fine, some level of the "author's voice" is expected to come through in the prose</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (ttdu) [inline interm_notebook_5_classify.ipynb:1]: <p>You had mentioned adding a section here about overfitting -- that so long as the testing/validation accuracies are both going up, we can be reasonably confident that we haven't overfit. </p><p> </p><p>I would add that here, before the reader launches into the next section. It helps to motivate these plots</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (ttdu) [inline interm_notebook_5_classify.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    ticids = ['120461526', '278779899', '139754153', '273418879', '52121469', '188580272', '394015919', '402104884']
</pre><p> </p><p>Without increasing the runtime too much, could we run through a long list of these and only display graphs when the probability of a flare is high?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (ttdu) [inline interm_notebook_5_classify.ipynb:1]: <p>Not sure if we were still planning on proceeding with this or not</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>
